### ISSUE TITLE
[Bug] lint-detekt error: LargeClass @ FenixApplication.kt

### ISSUE THREAD

**Issue by AaronMT** [mozilla-mobile/fenix#8867]
```Log
[task 2020-02-28T19:44:00.509Z] > Task :detekt
[task 2020-02-28T19:44:04.309Z] .........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
[task 2020-02-28T19:44:04.309Z] 
[task 2020-02-28T19:44:04.309Z] 505 kotlin files were analyzed.
[task 2020-02-28T19:44:04.309Z] 
[task 2020-02-28T19:44:04.309Z] Ruleset: comments
[task 2020-02-28T19:44:04.309Z] Ruleset: complexity - 20min debt
[task 2020-02-28T19:44:04.309Z] 	LargeClass - 201/200 - [FenixApplication] at /builds/worker/checkouts/src/app/src/main/java/org/mozilla/fenix/FenixApplication.kt:45:1
[task 2020-02-28T19:44:04.309Z] Ruleset: empty-blocks
[task 2020-02-28T19:44:04.309Z] Ruleset: exceptions
[task 2020-02-28T19:44:04.309Z] Ruleset: mozilla-detekt-rules
[task 2020-02-28T19:44:04.309Z] Ruleset: naming
[task 2020-02-28T19:44:04.309Z] Ruleset: performance
[task 2020-02-28T19:44:04.309Z] Ruleset: potential-bugs
[task 2020-02-28T19:44:04.309Z] Ruleset: style
[task 2020-02-28T19:44:04.309Z] 
[task 2020-02-28T19:44:04.309Z] Overall debt: 20min
[task 2020-02-28T19:44:04.309Z] 
[task 2020-02-28T19:44:04.309Z] Complexity Report:
[task 2020-02-28T19:44:04.309Z] 	- 56594 lines of code (loc)
[task 2020-02-28T19:44:04.309Z] 	- 42412 source lines of code (sloc)
[task 2020-02-28T19:44:04.309Z] 	- 28590 logical lines of code (lloc)
[task 2020-02-28T19:44:04.309Z] 	- 5680 comment lines of code (cloc)
[task 2020-02-28T19:44:04.309Z] 	- 5621 McCabe complexity (mcc)
[task 2020-02-28T19:44:04.309Z] 	- 1 number of total code smells
[task 2020-02-28T19:44:04.309Z] 	- 13 % comment source ratio
[task 2020-02-28T19:44:04.309Z] 	- 196 mcc per 1000 lloc
[task 2020-02-28T19:44:04.310Z] 	- 0 code smells per 1000 lloc
[task 2020-02-28T19:44:04.310Z] 
[task 2020-02-28T19:44:04.310Z] Project Statistics:
[task 2020-02-28T19:44:04.310Z] 	- number of properties: 3712
[task 2020-02-28T19:44:04.310Z] 	- number of functions: 3492
[task 2020-02-28T19:44:04.310Z] 	- number of classes: 906
[task 2020-02-28T19:44:04.310Z] 	- number of packages: 71
[task 2020-02-28T19:44:04.310Z] 	- number of kt files: 505
[task 2020-02-28T19:44:04.310Z] 
[task 2020-02-28T19:44:04.310Z] Successfully generated HTML report at /builds/worker/checkouts/src/build/reports/detekt.html
[task 2020-02-28T19:44:04.310Z] 
[task 2020-02-28T19:44:04.310Z] Build failed with 1 weighted issues (threshold defined was 0).
[task 2020-02-28T19:44:04.310Z] io.gitlab.arturbosch.detekt.cli.console.BuildFailure: Build failed with 1 weighted issues (threshold defined was 0).
[task 2020-02-28T19:44:04.310Z] 
[task 2020-02-28T19:44:04.310Z] > Task :detekt FAILED
[task 2020-02-28T19:44:04.408Z] 
[task 2020-02-28T19:44:04.408Z] FAILURE: Build failed with an exception.
[task 2020-02-28T19:44:04.408Z] 
[task 2020-02-28T19:44:04.408Z] * What went wrong:
[task 2020-02-28T19:44:04.409Z] Execution failed for task ':detekt'.
[task 2020-02-28T19:44:04.409Z] > MaxIssues or failThreshold count was reached.```

**Comment by AaronMT** [comment]
Should be fixed via https://github.com/mozilla-mobile/fenix/pull/8615/files

**Comment by ekager** [comment]
Fixed by #8615
